k8s/redis/deployment.yaml

```
@@ -0,0 +1,20 @@
+    matchLabels:
+      app: redis
+  template:
+    metadata:
+      labels:
+        app: redis
+    spec:
+      containers:
+      - name: redis
+        image: redis:7
+        ports:
+        - containerPort: 6379
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)